src/modules/ROOT/pages/changelog/202209/03.json

```diff
@@ -0,0 +1,139 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/a8d3b831db94538f6cfd84bd86fe35ec555b1bc7",
+        "https://github.com/cortezaproject/corteza/commit/b2c9de57a73c84aa16f255e8f7c281be91c76c34"
+      ]
+    }
+  ],
+
+  "changed": [
+    {
+      "what": [
+        "the resource list now uses the same component, item count and pagination and it is also enabled for resource lists that support it.",
+        "Now almost on every resource we show the total item count and we have the ability to go to the next page if there are items."
+      ],
+      "why": [
+        "because the look and feel of lists across {PRODUCT_NAME} needed to be unified.",
+        "Also for better user experience and to always know the correct number of how many records/items we have."
+      ],
+      "how": [
+        "by updating and using the reusable CResourceList component in corteza-vue.",
+        "That means we can control the look of all lists from one place."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-vue/commit/030109b86e3e2d35ec0f0104f5b279c0f33f7ebd", 
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/229bf12d9c6608f40bf49dfb6ef4ca68defa0875", 
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/c67daa7a60e1fb38a49a0df770aab6257e5f03bb", 
+        "https://github.com/cortezaproject/corteza-webapp-reporter/commit/06c07d8036f8ad3736c06bd69ae1ab23df095563",
+        "https://github.com/cortezaproject/corteza-webapp-workflow/commit/60334336292aca9ed4f581213d6d4bb300772986"
+      ]
+    },
+    {
+      "what": [
+        "the requirement to provide a handle when creating a resource is removed and this is now optional.",
+        "The change is applied in corteza-webapp-compose, corteza-webapp-admin, corteza-webapp-reporter and corteza-webapp-workflow."
+      ],
+      "why": [
+        "because providing a handle shouldn't be a necessity.",
+        "The resource can be created without it."
+      ],
+      "how": [
+        "by removing the requirement for the handle to be provided."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/cd86e854b76866eeb67d33753b1b736be0d1957a",
+        "https://github.com/cortezaproject/corteza-webapp-reporter/commit/cf21c47c17c1b76eefc461543236abdf04277cb0", 
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/456c896adaa7e3423c81d05a541f044b5acc9cc8", 
+        "https://github.com/cortezaproject/corteza-webapp-workflow/commit/9b88af07260977ea744cce8ff7f54185ad9bd5d3", 
+        "https://github.com/cortezaproject/corteza-locale/commit/24b0dda5d434b8f8f0df55d98d6695171533d81c", 
+        "https://github.com/cortezaproject/corteza-vue/commit/34f261acc5297654eb372b8e804a7fd37e7b0d0e",
+        "https://github.com/cortezaproject/corteza-js/commit/3a90a1002074e31017b2ae22e5d74404283ea3e2" 
+      ]
+    },
+    {
```

Comment:
Isn't this an addition? A change would indicate that something doesn't work the same as it did before; for example changing the default to be 0 instead of 1